Error when receiving the private message

[DATA CORRUPTED]

```
[WARN] [06/22/2014 15:50:42.473] [Smack Listener Processor (0)] [akka://CodingteamSystem/user/core/jabber] I had to repeat message "????l??u: ? ??????? ???? ??????!" because server hates me
[INFO] [06/22/2014 15:50:42.473] [CodingteamSystem-akka.actor.default-dispatcher-772] [akka://CodingteamSystem/user/core/jabber/privateHandler] Private message: <codingteam@conference.jabber.ru> ????l??u: ? ??????? ???? ??????!
[ERROR] [06/22/2014 15:50:42.474] [CodingteamSystem-akka.actor.default-dispatcher-772] [akka://CodingteamSystem/user/core/$c] None.get
java.util.NoSuchElementException: None.get
        at scala.None$.get(Option.scala:313)
        at scala.None$.get(Option.scala:311)
        at ru.org.codingteam.horta.plugins.log.LogPlugin.processMessage(LogPlugin.scala:46)
        at ru.org.codingteam.horta.plugins.MessageProcessor$$anonfun$receive$1.applyOrElse(MessageProcessor.scala:12)
        at scala.runtime.AbstractPartialFunction$mcVL$sp.apply$mcVL$sp(AbstractPartialFunction.scala:33)
        at scala.runtime.AbstractPartialFunction$mcVL$sp.apply(AbstractPartialFunction.scala:33)
        at scala.runtime.AbstractPartialFunction$mcVL$sp.apply(AbstractPartialFunction.scala:25)
        at ru.org.codingteam.horta.plugins.CommandProcessor$$anonfun$receive$1.applyOrElse(CommandProcessor.scala:12)
        at akka.actor.ActorCell.receiveMessage(ActorCell.scala:498)
        at akka.actor.ActorCell.invoke(ActorCell.scala:456)
        at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:237)
        at akka.dispatch.Mailbox.run(Mailbox.scala:219)
        at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386)
        at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
        at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339)
        at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
        at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

[DATA CORRUPTED]

^Wlease contact level █ administrator

--- a/src/main/scala/ru/org/codingteam/horta/plugins/log/LogPlugin.scala
+++ b/src/main/scala/ru/org/codingteam/horta/plugins/log/LogPlugin.scala
@@ -47,7 +47,7 @@ class LogPlugin extends BasePlugin with ParticipantProcessor with MessageProcess
   }
 
   override protected def processMessage(time: DateTime, credential: Credential, message: String) {
-    saveLogMessage(time, credential.roomId.get, credential.name, MessageType, message)
+    credential.roomId.map(roomId => saveLogMessage(time, roomId, credential.name, MessageType, message))
   }
 
   def processCommand(credential: Credential,

